### ISSUE TITLE
Big timelapses in my tracker-map

### ISSUE THREAD

**Issue by Reiker** [petewarden/iPhoneTracker#15]
Hi! 
I have big timelapses in my map after loading iPhonetracker.app, there can be weeks between "timesteps" and i have no time stamps at all...   Is there any way to set the resolution, lets say i want to import every single day(the last year or so) on the map? 

The example video(amtrak) on the homepage had a lot more information imported and shorter "timesteps". How can i get a more detailed map of my own?

Would really appreciate the help, and thank you for the awesome work!

/Reiker

**Comment by stoner** [comment]
The lapses are probably due to all the cell towers in your local area being cached on to your phone, and if you don't travel to an new area then you will not get new entries in the cell tower location cache.

**Comment by petewarden** [comment]
There is actually some deliberate inaccuracy in the displayed data, since we
wanted to ensure it wouldn't be too useful for snoops. We constrain all the
points to lie on 1/100th of a degree grid points, and only animate
week-by-week.

It is possible for code-literate people to 'fix' that, but we wanted the
easy-to-find binary to be less useful for detailed views.

**Comment by uhuznaa** [comment]
You should really, really remove the deliberate inaccuracy thing. The only thing this does is making people assume that the data is much more accurate in reality while in fact it isn't. These are locations of cell towers, not of the iPhone. I fixed that in the app and the locations are often miles (sometimes tens of miles) away from the real location of the phone. 

You actually generate paranoia by fuzzing the locations. Whenever someone looks at the raw data and finds that the data is not reporting the location of the iPhone and tries to tell others of this, they tell him that your app is not reporting the true locations but the iPhone records your true location. You're creating fear, uncertainty and doubt with this "feature".

**Comment by stoner** [comment]
I agree.  I can hardly think of a more dramatic massaging of the Cell Location table.  Here is my current interpretation of the data, but when ones inspects the WiFi data (see at end) then the oringnal concerns of the iPhoneTracket team holds true.

1/. It is the cell towers, plus areas, as downloaded by the iPhone, from a apple database.  The areas are  the records where the CI is -1.  Since I have managed to get a couple of iPhones worth of data and it seems for a visual inspection that the locations of the cell's that are in both samples are exactly the same.

2/ Also the cell towers location match towers as reported on 
http://realtimeblog.free.fr/maps/with_google_earth.php5?mcc=234 and 
http://www.sitefinder.ofcom.org.uk/

I would want to do a complete comparison between the two data sets, and visually inspect the suspected micro cells.  These are the reason for so many points in the database, these cells only cover a few hundred meters diameter. 
If anyone in UK SE on O2 could export the follow as KML SELECT  MCC || '_' || MNC || '_' || LAC|| '_' || CI as ID,   Latitude , Longitude  from the cells, and dump in the KML fork, with name and description set to this id that would be lovely ;

2/. The cell location is in the database once, and once only.
The primary key on the database ensures this.

3/.The first time you iphone 'sees' a cell it is recorded.
Therefore your regular journeys  will be recorded once, so the first day you have your new iPhone it will track/cache most of the day/week.
I have changed the application to show the a weeks worth journeyin second timestamps and this demonstrates it.
Only a few points for weeks, and then a trip, loads of points.

4/. The timestamp is the requested time of for the data, or received from the database, not investigated which.
Batches of Cells location share exactly the same timestamp.  Individual cells don’t seem to be requested on their own.  
When data is disabled it seems that you can get massive 100KM wide areas at once (maybe WiFi only).

So in conclusion it seems that the tracking is less than first reported, but...

WiFi

There are lots of WiFi points, smaller area covered and these are turned on an off at different times durning the day, new ones installed etc, so an iPhone will detect these even when on a regular run.
I’ve not investigated the easy in which peoples WiFi MAC address and physical location  can be determined.  This may be the greatest chance of harm to people.
Also is the table the result of the iPhone using the GPS and detecting WiFi, downloaded from apple, or both?
